mlflow/utils/file_utils.py

```diff
@@ -377,7 +377,7 @@ def __init__(self, chdr=False, remove_on_exit=True):
         self._remove = remove_on_exit
 
     def __enter__(self):
-        self._path = os.path.abspath(tempfile.mkdtemp())
+        self._path = os.path.abspath(create_tmp_dir())
```

Comment:
Curious why we need this change.

Reply:
https://databricks.atlassian.net/browse/ES-836443
Seems on databricks we should use local_disk0

Reply:
on databricks, `/tmp` storage space cannot be scaled, but `/local_disk0` is designed to be scalable. we should save large temporary files to `/local_disk0`

Reply:
For small files, using `/tmp` is fine, right?

Reply:
We definitely should not be using /tmp for these very large LLMs. Is there a concern that you have @harupy for using ebs scalable storage within Databricks for everything? It will likely:
1. Be a simpler implementation 
2. Not require size estimation logic that, for LLMs, will require extensive recursive directory inspection (there are a LOT of files in these repos) 
3. Prevent any potential issues with other usages of the /tmp swap space on the Spark Driver

Reply:
To make logic simple , i think we can always use  /local_disk0. It don't have performance difference with /tmp